src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -87,8 +89,10 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
 
       profileResponse pipeTo context.parent
 
-    case UpdateNIHLink(userInfo: UserInfo, nihLink: NIHLink) =>
+    case UpdateNIHLinkAndSyncSelf(userInfo: UserInfo, nihLink: NIHLink) =>
       val parent = context.parent
+      // Complete syncWhitelist and ignore as neither success nor failure are useful to the client
+      ask(self, ProfileClient.SyncSelf(userInfo))(1.minute)
```

Comment:
From what I understand (http://doc.akka.io/docs/akka/current/scala/futures.html), `ask` is the non-blocking alternative to `Await`. I am pretty sure we do *not* want to introduce a blocking call like Await here.